ts/Data/DataTable.ts

```diff
@@ -1601,8 +1680,18 @@ class DataTable implements DataEvent.Emitter {
         });
     }
 
+    private removeKeyIdColumn(columnNamesOrAliases: Array<string>): void {
+        if (this.rowKeysId) {
+            const pos = columnNamesOrAliases.indexOf(this.rowKeysId);
+            if (pos !== -1) {
+                // Alays the last column
+                columnNamesOrAliases.pop();
```

Comment:
"Alays" -> "Always" - but besides the typo, the implementation does not cover `DataTable.setColumns` and `DataTable.deleteColumns` yet. So the statement is not true until all alteration functions of DataTable deal with the hidden rowKeysId column.